ShopLens/ShopLensForms/Views/MainForm.cs

```diff
@@ -11,21 +11,19 @@ public partial class ShopLens : Form
     {
         private FilterInfoCollection _captureDevices;
         private VideoCaptureDevice _videoSource;
-        private MainController _mainController;
+        public MainController MainController { get; set;}
```

Comment:
Since MainController is public, do we still need get, set methods?

Reply:
Imma make it private then.